src/backend/src/prisma/schema.prisma

```diff
@@ -636,3 +638,29 @@ model Meeting {
   team         Team?  @relation(fields: [teamId], references: [teamId])
   teamId       String
 }
+
+model Blocked_By_Info {
+  blockedByInfoId       String                @id @default(uuid())
+  stage                 Work_Package_Stage?
+  name                  String
+  workPackageTemplate   Work_Package_Template @relation(fields: [workPackageTemplateId], references: [workPackageTemplateId])
+  workPackageTemplateId String
+}
+
+model Work_Package_Template {
+  workPackageTemplateId String              @id @default(uuid())
+  templateName          String?
+  templateNotes         String?
```

Comment:
I don't think these two are supposed to be optional

Reply:
^^